### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by thomasBeckSecib**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by ptalrejagithub** [comment]
Thank you for your submission! Our team will review your files and will get back to you within two business days.

File: certified-connectors/Secib/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# SECIB
+SECIB provides a REST API which can be used to manage your SECIB's data (such as your case files, your calendar, your documents, etc).
+
+The SECIB connector uses that REST API to allow SECIB's users to automate their workflow. For instance, you can trigger a worklflow when a new case file is created in SECIB neo and then process this case file before sending an email notification with the case file's information to other SECIB's users.
+
+
```
Comment (sriyen-msft) [inline certified-connectors/Secib/README.md:6]: @thomasBeckSecib Please remove extra spacing in all the artifacts.

File: certified-connectors/Secib/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Publisher: Publisher's Name
```
Comment (sriyen-msft) [inline certified-connectors/Secib/README.md:8]: @thomasBeckSecib This section is not required in readme.md, Please remove it.

File: certified-connectors/Secib/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,755 @@
...
+                    }
+                  },
+                  "description": "result"
+                }
+              }
+            }
+          },
+          "310": {
+            "description": "Too Many Redirects",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/Secib/apiDefinition.swagger.json:168]: @thomasBeckSecib Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response.

File: certified-connectors/Secib/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,755 @@
...
+                "dossierId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Identifier of the case file.",
+                  "title": "Identifier"
+                },
+                "code": {
+                  "type": "string",
+                  "description": "Number of the case file.",
+                  "title": "Case file n\u00b0",
```
Comment (sriyen-msft) [inline certified-connectors/Secib/apiDefinition.swagger.json:221]: @thomasBeckSecib Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

File: certified-connectors/Secib/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,755 @@
...
+                    }
+                  },
+                  "description": "Lead attorney of the case file"
+                }
+              },
+              "description": "result"
+            }
+          }
+        },
+        "description": "CreatedDossierWebhookResponse"
```
Comment (sriyen-msft) [inline certified-connectors/Secib/apiDefinition.swagger.json:303]: @thomasBeckSecib All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

File: certified-connectors/Secib/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,755 @@
...
+    {
+      "oauth2_auth": [
+        "{API Permissions of the connector in Azure App Registrations}"
+      ]
+    }
+  ],
+  "tags": [
+    {
+      "name": "Dossiers",
+      "description": ""
```
Comment (sriyen-msft) [inline certified-connectors/Secib/apiDefinition.swagger.json:726]: @thomasBeckSecib Empty description can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Secib/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "{Application (client) ID of the connector in Azure App Registrations}",
+          "scopes": [
+            "{API Permissions of the connector in Azure App Registrations}"
```
Comment (sriyen-msft) [inline certified-connectors/Secib/apiProperties.json:10]: @thomasBeckSecib Please make sure you are providing valid scopes while submitting the connector into ISV Studio for certification.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Secib/README.md (hunk 1 above)
Comment (thomasBeckSecib) [reply certified-connectors/Secib/README.md]: I removed extra spacing in all artifacts.

File: certified-connectors/Secib/README.md (hunk 2 above)
Comment (thomasBeckSecib) [reply certified-connectors/Secib/README.md]: I removed this section.

File: certified-connectors/Secib/apiDefinition.swagger.json (hunk 3 above)
Comment (thomasBeckSecib) [reply certified-connectors/Secib/apiDefinition.swagger.json]: Now, every response have a specified schema (but i causes warnings in the paconn validate step)

File: certified-connectors/Secib/apiDefinition.swagger.json (hunk 4 above)
Comment (thomasBeckSecib) [reply certified-connectors/Secib/apiDefinition.swagger.json]: Every unicodes characters are now removed from artifacts.

File: certified-connectors/Secib/apiDefinition.swagger.json (hunk 5 above)
Comment (thomasBeckSecib) [reply certified-connectors/Secib/apiDefinition.swagger.json]: Every description are complete now.

File: certified-connectors/Secib/apiDefinition.swagger.json (hunk 6 above)
Comment (thomasBeckSecib) [reply certified-connectors/Secib/apiDefinition.swagger.json]: I removed empty descriptions.

File: certified-connectors/Secib/apiProperties.json (hunk 7 above)
Comment (thomasBeckSecib) [reply certified-connectors/Secib/apiProperties.json]: Every scopes are now provided in connector artifacts but without the clientId that composed some of them.

File: certified-connectors/Secib/apiProperties.json [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "{Application (client) ID of the connector in Azure App Registrations}",
+          "scopes": [
+            "User.Read",
+            "User.ReadBasic.All",
+            "api://{Application (client) ID of the web API in Azure App Registrations}/create",
```
Comment (sriyen-msft) [inline certified-connectors/Secib/apiProperties.json:12]: @thomasBeckSecib Are you sure these scope values are valid as no scope value depends on its client or application id.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Secib/apiProperties.json (hunk 8 above)
Comment (thomasBeckSecib) [reply certified-connectors/Secib/apiProperties.json]: You were right, the clientId is not necessary. The scopes are now valid.

File: certified-connectors/Secib/apiProperties.json [hunk 9]
```diff
@@ -0,0 +1,55 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "{Application (client) ID of the connector in Azure App Registrations}",
```
[the remark this answers is not in the document]
Comment (thomasBeckSecib) [reply certified-connectors/Secib/apiProperties.json]: I replaced dummy text to maintain consistency with other connectors.

File: certified-connectors/Secib/apiProperties.json (hunk 7 above)
Comment (thomasBeckSecib) [reply certified-connectors/Secib/apiProperties.json]: Scopes are valid now.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**